tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -390,6 +391,26 @@ def test_check_sfputil_eeprom(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
             assert parsed_eeprom[intf] == "SFP EEPROM detected"
 
 
+@pytest.mark.device_type('physical')
+def test_check_sfputil_eeprom_hexdump(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
+                                      enum_frontend_asic_index, conn_graph_facts, xcvr_skip_list):
+    """
+    @summary: Check eeprom hexdump using 'sfputil show eeprom-hexdump'
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    global ans_host
```

Comment:
why do we need it as a global? using gloabls in general should be avoided, unless there's a very good justification for that